ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -86,6 +87,21 @@ public void connect(final String url, @Nullable final ReadableArray protocols, @
       }
     }
 
+    if (protocols != null && protocols.size() > 0) {
+      StringBuilder protocolsValue = new StringBuilder("");
+      for (int i = 0; i < protocols.size(); i++) {
+        String v = protocols.getString(i).trim();
+        if (!TextUtils.isEmpty(v) && !v.contains(",")) {
```

Comment:
thanks, it can be avoided to import additional `TextUtils`.

**Just a side note:**

`String.isEmpty()`
only checks length of string and introduced since API level 9 (Android 2.3)
so it maybe caused some crashes on earlier versions of Android.

`TextUtils.isEmpty()`
checks both length of string and null.
is available since API level 1